brightchain-lib/src/lib/documents/document.ts

```
@@ -0,0 +1,314 @@
+    if (schema) {
+      this.validateSchema(this.data);
+    }
+    this.instanceMethods = instanceMethods;
+    this.staticMethods = staticMethods;
+    Object.assign(this, instanceMethods);
+    Object.assign(Document, staticMethods);
+  }
+
+  public static initialize(
+    blockService: BlockService,
+    memberService?: MemberCblService,
```

ESLint

[error] 'memberService' is defined but never used.

(@typescript-eslint/no-unused-vars)